src/macaron/slsa_analyzer/ci_service/base_ci_service.py

```diff
@@ -119,10 +119,10 @@ def extract_all_bash(self, callgraph: CallGraph, macaron_path: str = "") -> Iter
         """
         raise NotImplementedError
 
-    def has_kws_in_config(self, kws: list, repo_path: str) -> str:
-        """Check the content of a file for any build keywords.
+    def has_kws_in_config(self, kws: list, repo_path: str) -> tuple[str, str]:
+        """Check the content of all config file in a repository for any build keywords.
```

Comment:
```suggestion
        """Check the content of all config files in a repository for any build keywords.
```

Reply:
Fixed in https://github.com/oracle-samples/macaron/pull/136/commits/4601d2ecbfb49d55cb4d4f1dfeacc0305ffecb71.